### ISSUE TITLE
Processed tag count is off when multiple paths are given.

### ISSUE THREAD

**Issue by ianmacd** [henkelis/sonospy#63]
See output below. The processed tags count should not be 29.

```
$ ./scan.py -d /opt/local/home/ianmacd/src/sonospy/sonospy/sonos.sqlite /media/audio/altered_images /media/audio/throwing_muses          
./gettags.py -d /opt/local/home/ianmacd/src/sonospy/sonospy/sonos.sqlite /media/audio/altered_images /media/audio/throwing_muses
Database: /opt/local/home/ianmacd/src/sonospy/sonospy/sonos.sqlite
Scanning: /media/audio/altered_images
Scannumber: 48
New file found: 01_intro_happy_birthday.flac, /media/audio/altered_images/happy_birthday
New file found: 02_love_and_kisses_.flac, /media/audio/altered_images/happy_birthday
New file found: 03_real_toys_.flac, /media/audio/altered_images/happy_birthday
New file found: 04_idols_.flac, /media/audio/altered_images/happy_birthday
New file found: 05_legionnaire_.flac, /media/audio/altered_images/happy_birthday
New file found: 06_faithless_.flac, /media/audio/altered_images/happy_birthday
New file found: 07_beckoning_strings.flac, /media/audio/altered_images/happy_birthday
New file found: 08_happy_birthday.flac, /media/audio/altered_images/happy_birthday
New file found: 09_midnight_.flac, /media/audio/altered_images/happy_birthday
New file found: 10_a_days_wait.flac, /media/audio/altered_images/happy_birthday
New file found: 11_leave_me_alone_.flac, /media/audio/altered_images/happy_birthday
New file found: 12_insects_.flac, /media/audio/altered_images/happy_birthday
New file found: 13_outro_happy_birthday.flac, /media/audio/altered_images/happy_birthday
New file found: 14_dead_pop_stars.flac, /media/audio/altered_images/happy_birthday
New file found: 15_sentimental.flac, /media/audio/altered_images/happy_birthday
New file found: 16_who_cares.flac, /media/audio/altered_images/happy_birthday
New file found: 17_happy_birthday_dance_mix.flac, /media/audio/altered_images/happy_birthday
New file found: 18_so_we_go_whispering.flac, /media/audio/altered_images/happy_birthday
New file found: 19_jeepster.flac, /media/audio/altered_images/happy_birthday
Scanning: /media/audio/throwing_muses
Scannumber: 49
New file found: 08_solar_dip.flac, /media/audio/throwing_muses/throwing_muses
Existing file not found: 08_solardip.flac, /media/audio/throwing_muses/throwing_muses
./movetags.py -s /opt/local/home/ianmacd/src/sonospy/sonospy/sonos.sqlite -d /opt/local/home/ianmacd/src/sonospy/sonospy/sonos.sqlite
Processing tags
Tags processed: 29
```

**Comment by henkelis** [comment]
How many tracks are there on the two paths?

**Comment by ianmacd** [comment]
Many more than 29, but I was assuming that this count should be the sum of new files, modified files and deleted files.

Is my assumption incorrect?

**Comment by henkelis** [comment]
The count is the number of tag/virtual/playlist updates that have been 
found by gettags. At the moment that is per path, I'll change it to 
write once per path. As you processed 2 paths, it looks like the first 
counted up to 19, then the second up to 2, but the 2 overwrote the 1 
(hence the 29).
That text gets overwritten by the 'tags processed' message, but the 
count (unfortunately) remains visible.

I've fixed it in the soon-to-be-released albumsort version as I was working on that module anyway.

**Comment by ianmacd** [comment]
Oh, so it's just a cosmetic thing, i.e. a CR without a LF?

No worries, then.

**Comment by henkelis** [comment]
Fixed in 15298f7
